src/main/java/com/epam/spring/boot/cargodeliverysystem/repository/ReceiptRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.spring.boot.cargodeliverysystem.repository;
+
+import com.epam.spring.boot.cargodeliverysystem.model.Receipt;
+
+import java.util.List;
+
+public interface ReceiptRepository {
+
+    List<Receipt> getAllReceipts();
+
+    Receipt getReceipt(long id);
```

Comment:
I would use getReceiptById() instead, it has more readability

Reply:
renamed